integration-test/transform-tests/config/rollup.config.cjs.js

```diff
@@ -12,7 +12,7 @@ export default [
     plugins: [
       replace({
         delimiters: ['', ''],
-        "import Nat from '../src/index';": "import Nat from '@agoric/nat';",
+        "import { isNat, Nat } from '../src/index';": "import { isNat, Nat } from '@agoric/nat';",
```

Comment:
Could this be more robust?

```suggestion
        "'../src/index'": "'@agoric/nat'",
```

Reply:
Yup, that is better!